Dockerfile

```diff
@@ -9,10 +9,12 @@ RUN apt-get update && \
       nginx \
       zlib1g-dev \
       libzip-dev \
-      libmemcached-dev && \
+      libmemcached-dev \
+      libicu-dev && \
```

Comment:
Is this `libicu-dev` really needed? Just making sure. Don't like bloating the deps.

Reply:
Required by `ext-intl`, as noted above.